client/src/pages/index.jsx

```diff
@@ -118,7 +116,9 @@ const IndexPage = ({ search }) => {
   }
 
   useEffect(() => {
-    dispatch(cmi5Start());
+    if (Cmi5.isCmiAvailable) {
+      Cmi5.instance.initialize();
```

Comment:
you probably want to avoid the potential promise rejection here by catching exceptions, e.g.

```typescript
  Cmi5.instance.initialize().
  catch(e => {
    console.error(e)
  })
```